modules/managing-feature-flags.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * configuration/managing-preview-features.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="managing-feature-flags_{context}"]
+= Managing Feature Flags
+
+== Prerequisites
+* You have access to the environment where the {product-title-short} component is deployed.
+* You have permission to modify environment variables.
+* You understand that the Technological Preview features might be incomplete and are provided with limited support.
+* You know if the feature flag needs to be configured before the deployment. Check the installation manifests to see if they use the required flag.
+
+.Procedure
+
+. Identify the environment variable name associated with the feature flag.
+Consult the release notes or the `/v1/featureflags` API endpoint to identify the flag for the feature you want to enable or disable.
+
+. Modify the feature flag by completing one of the following actions:
+* To enable a feature, configure the environment variable associated with the flag by setting its value to `true`: either directly on the k8s deployment, or via the helm chart, or via the operator CR.
+* To disable a feature, set the environment variable associated with the flag to `false`.
+
+. After the application is restarted or redeployed, verify that the feature has been enabled or disabled by completing the following steps:
+
+* Check the output of the `/v1/featureflags` API endpoint.
+* Check the application functionality related to the feature.
+* Review logs or monitoring tools for any errors or confirmation messages.
+
+== Best Practices
```

Comment:
```suggestion
== Best practices
```
https://redhat-documentation.github.io/supplementary-style-guide/#titles-and-headings